blog/templates/blog/post.html

```diff
@@ -2,9 +2,31 @@
 {% block title %}PolarBear -- Postpage{% endblock %}
 {% block content %}
 
-<form action="{% url 'post' %}" method="post">
+
+<form action="{% url 'post' %}" method="post" enctype="multipart/form-data" >
   {% csrf_token %}
   {{ form.as_p }}
   <input type="submit" value="Post!">
 </form>
+
+<script>
+	// This purpose of this function is 
+	// to hide image field when we choose to use Markdown
+	// and display image field when we use plain text
+
+	function autohide(){
+		console.log(document.getElementById('id_content_type').selectedIndex)
+		if (document.getElementById('id_content_type').selectedIndex){
+			// This is Image Field (maybe a stupid way to locate it)
+			// hide it
```

Comment:
you can give the form element an id in django with `attrs={'id':'id'}` as an arugment to the `imageField` (I think).

example:
```python
image = forms.ImageField(label = 'Choose an Image', required=False, widget=forms.FileInput(attrs={'id':'id'}))
```

Reply:
I tried this way, It can add an id for input field only, but cannot add an ID for labels.

Reply:
ok